### PR TITLE
DOC test fix - add to env

### PR DESCRIPTION
**Pull request by GreatRSingh**
## Description

Fix #(issue)

## Type of change

Please check the option that is related to your PR.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
  - In this case, we recommend to discuss your modification on GitHub issues before creating the PR
- [ ] Documentations (modification for documents)

## Checklist

- [ ] My code follows [the style guidelines of this project](https://deepchem.readthedocs.io/en/latest/development_guide/coding.html)
  - [ ] Run `yapf -i <modified file>` and check no errors (**yapf version must be  0.32.0**)
  - [ ] Run `mypy -p deepchem` and check no errors
  - [ ] Run `flake8 <modified file> --count` and check no errors
  - [ ] Run `python -m doctest <modified file>` and check no errors
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New unit tests pass locally with my changes
- [ ] I have checked my code and corrected any misspellings

### CODE REVIEW

**Review by rbharath — COMMENTED** [review]
@GreatRSingh Are there are any tests you can confirm this fixes?

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -163,7 +163,7 @@ jobs:
     - name: Doctest Linux
       if: ${{ (runner.os == 'Linux') && always() }}
       shell: bash -l {0}
-      run: DGLBACKEND=pytorch pytest -v --ignore-glob='deepchem/**/test*.py' --doctest-modules deepchem --doctest-continue-on-failure
+      run: DGLBACKEND=pytorch pytest -v --ignore-glob='deepchem/**/test*.py' --ignore-glob='deepchem/models/jax_models/*' --doctest-modules deepchem --doctest-continue-on-failure
```
Comment (rbharath) [inline .github/workflows/test.yml:166]: Why are we removing the jax tests?

**Comment by GreatRSingh** [comment]
> @GreatRSingh Are there are any tests you can confirm this fixes?

https://github.com/deepchem/deepchem/actions/runs/7242816554/job/19728749079?pr=3731

`Unit Tests -> Doctest Linux` was unable to run, not that test runs.

File: .github/workflows/test.yml (hunk 1 above)
Comment (GreatRSingh) [reply .github/workflows/test.yml:166]: jax is not part of that environment.
Let's try to fix that in another pr. will be simpler in a separate pr.

**Review by rbharath — APPROVED** [review]
LGTM

**Pull request merged**